packages/services/src/services/CloudAccountAuth/CloudAccountAuth.js

```diff
@@ -13,7 +13,7 @@ import Cookies from 'js-cookie';
  * @type {string}
  * @private
  */
-const _cookieName = 'com.ibm.cloud.iam.LoggedIn.manual';
+const _cookieName = 'com.ibm.cloud.iam.LoggedIn.prod';
```

Comment:
@proeung something to consider for the future, perhaps make this configurable via environment variable, and the default is the prod cookie name?

Reply:
Yes! I agree 💯 

I'll create a feature request issue to track this improvement.